docs/tutorials/send_receive.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Sending & receiving message over a deployed broker"  
+description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on OpenShift"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "tutorials"
+weight: 110
+toc: true
+---
+
+### Prerequisite
+
+Before you start, you need to have access to a running Kubernetes cluster
+environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance
+running on your laptop will do fine.
+
+### Step 1 - Deploy the operator
+
+#### create the namespace
+
+```shell script
+$ kubectl create namespace activemq-artemis-operator
```

Comment:
The user could use the following command to set the namespace of his current context:
```
kubectl config set-context --current --namespace=activemq-artemis-operator
```